bin/dune_init.ml

```diff
@@ -245,20 +245,27 @@ module Component = struct
     module Common = struct
       type t =
         { name : Dune_lang.Atom.t
+        ; public : Public_name.t option
         ; libraries : Dune_lang.Atom.t list
         ; pps : Dune_lang.Atom.t list
         }
+
+      let opam_package_name common =
```

Comment:
How about just `package_name`? We don't have any other types of packages in dune.

Reply:
Sure! I've renamed it :)